docs/export.md

```diff
@@ -92,6 +92,8 @@ optional arguments:
   --model-path MODEL_PATH
                         Path to the folder containing model related files.
                         Signature file is required.
+  --from-type {mxnet,onnx}
```

Comment:
What happens if it is not provided?
Let's follow the convention in the other arg descriptions and say: "If not provided, the tool assumes the format is MXNet.

Reply:
```
--from-type {mxnet,onnx}
  Optional model source type. By default it assumes mxnet unless otherwise specified.
```

Reply:
Hmm. Thinking about this again, I actually think the default behavior should be to *infer* the model-format from the file extension if possible.